commands/bench-all/lib/bench-all-pallet.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+shopt -s inherit_errexit
+shopt -s globstar
+
+. "$(dirname "${BASH_SOURCE[0]}")/../../utils.sh"
+. "$(dirname "${BASH_SOURCE[0]}")/../../cmd_runner.sh"
+
+get_arg required --pallet "$@"
+PALLET="${out:-""}"
+
+REPO_NAME="$(basename "$PWD")"
+BASE_COMMAND="$(dirname "${BASH_SOURCE[0]}")/../../bench/bench.sh --noexit=true --subcommand=pallet"
+WEIGHT_FILE_PATHS=()
+
+while IFS= read -r -d '' f; do
+  FILE=$(echo $f | sed 's|^\./||g')
+  WEIGHT_FILE_PATHS+=("$FILE")
+done < <(find . -type f -name "${PALLET}.rs" -path "**/weights/*" -print0)
```

Comment:
I think either `IFS= `, or `-print0` is redundant. If you're going with null-byte separation, you probably want to use `-r -d $'\0'`, instead of `IFS= `: 

Like this
```bash
find . -type f -name "${PALLET}.rs" -path "**/weights/*" -print0 \
| sed 's|^\./||g' \
| while -r -d $'\0' f; do
  WEIGHT_FILE_PATHS+=("$f")
done
```, or

```bash
while -r -d $'\0' f; do
  WEIGHT_FILE_PATHS+=("$f")
done < <(find . -type f -name "${PALLET}.rs" -path "**/weights/*" -print0 | sed 's|^\./||g')
```

However, if we don't really care for paths with spaces (and it seems that we don't), it can be simplified to 
```bash
WEIGHT_FILE_PATHS+=( $(find . -type f -name "${PALLET}.rs" -path "**/weights/*" | sed 's|^\./||g') )
```

Reply:
`while -r -d $'\0' f;` -> `-r: command not found`

and `WEIGHT_FILE_PATHS+=( $(find . -type f -name "${PALLET}.rs" -path "**/weights/*" | sed 's|^\./||g') )` doesn't work, because in this this way it doesn't allow then to add substrate/trappist individually... not sure why, but `WEIGHT_FILE_PATHS+=("$SUBSTRATE_PALLET_PATH")` just ignored, that's why while-do and adding them 1 by 1 works better

Reply:
it's `while read`, of course, sorry 🤦‍♂️
updated the comment

Reply:
ah, that was the other problem - &>/dev/null on ls... so one-liner works actually - 

```bash
WEIGHT_FILE_PATHS+=( $(find . -type f -name "${PALLET}.rs" -path "**/weights/*" | sed 's|^\./||g') )
```